File: 2st/Riyeon/배열의 길이에 따라 다른 연산하기.java
class Solution {
    public int[] solution(int[] arr, int n) {
        int l = arr.length;
        for(int i=0; i<arr.length; i++){
            if((l%2==0 && i%2==1)||(l%2==1 && i%2==0))
               arr[i] += n;
        }
        return arr;
    }
}

File: 2st/Riyeon/배열의 길이에 따라 다른 연산하기.py
def solution(arr, n):
    arrlen = len(arr) 
    for i in range(len(arr)):
        if i % 2 == 0 and arrlen % 2 == 1:
            arr[i] += n
        elif i % 2 == 1 and arrlen % 2 == 0:
            arr[i] += n
    return arr